resources/python/README.md

```diff
@@ -8,14 +8,23 @@ Officially Supported Python Versions:
 - Mac 
   - 3.7.16
   - 3.9.16
+  - 3.10.13
 - Windows 
   - 3.7.9
   - 3.9.13
+  - 3.10.11
```

Comment:
Stupid question: why don't we use the same minor version for Windows?

Reply:
It is not available for Windows: https://www.python.org/downloads/windows/